src/main/java/ch/njol/skript/effects/EffBlockUpdate.java

```diff
@@ -0,0 +1,68 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.block.Block;
+import org.bukkit.block.BlockState;
+import org.bukkit.block.data.BlockData;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Update Block")
+@Description({
+	"Updates the blocks to a selected block",
+	"`force`: Will force the update of the block",
```

Comment:
This `force` doesn't seem to be in the syntax anywhere. Did you mean to remove it from your description?